File: vb.net_introductory/Sample1.vb
Public class Sample1

    Private fsComment As String

    Private Sub BtnDecision_Click(sender As Object, e As EventArgs) Handles btnDecision.Click


    fsComment = "テスト"

        NameChange()

    End Sub

    Private Sub NameChange()

        txtComment.Text = fsComment
    End Sub

End Class

File: vb.net_introductory/Sample1.cs
public class Sample1

    private Sub BtnDecision_Click(sender As Object, e As EventArgs) Handles btnDecision.Click

    Dim lsComment As String

    lsComment = "テスト"

        txtComment.Text = lsComment

    End Sub

    Private Sub NameChange()

        txtComment.Text = lsComment
    End Sub

End Class